### ISSUE TITLE
Audio Mix Issue

### ISSUE THREAD

**Issue by chancebeyer1** [stephengpope/no-code-architects-toolkit#26]
Hello,

When trying the api with "output_length": "audio" it errors out. However if I just change that to "video" it works. Any suggestions?

Thanks

**Comment by stephengpope** [comment]
thanks for letting me, have you tried the new build? I pushed an update last night that should fix it.

https://hub.docker.com/layers/stephengpope/no-code-architects-toolkit/latest/images/sha256-66f8a0ea353116b9e0bf642deff0ce58f6e788dae2f0b57df67f5ba5a54061c0?context=repo

**Comment by chancebeyer1** [comment]
Ahhhh I am testing with my Google Cloud Console Deploy that I deployed before last night. Does Google not auto deploy when code changes?

**Comment by chancebeyer1** [comment]
Okay I redeployed my Cloud Run. Now getting different error:

API response: 

{
    "build_number": 15,
    "code": 202,
    "id": "unique_request_id",
    "job_id": "c2fe3ef3-3106-4859-8917-b01b0631b31a",
    "max_queue_length": "unlimited",
    "message": "processing",
    "pid": 3,
    "queue_id": 68494831512064,
    "queue_length": 1
}

Webhook Response:

![image](https://github.com/user-attachments/assets/9510704d-6011-49a7-b760-90cdc2c75abe)

**Comment by stephengpope** [comment]
no, GCP won't auto deploy, but we'll figure out a way to make sure that happens.

thanks for the details, we'll check it out shortly.

**Comment by jhogansjr** [comment]
@chancebeyer1  Did you ever get it working? I got this error if my when using "audio" and the audio is longer than the video.

**Comment by lourenzo-a** [comment]
Hey @stephengpope asking gpt for a help, it says:

**Misplaced -stream_loop parameter:**

The -stream_loop parameter must come before the file it applies to.
Solution: Rearrange the command to place "-stream_loop -1" before /tmp/ee89f915-5d36-40d1-98c1-7c580f74bfd0.mp4 (video file).

**Comment by stephengpope** [comment]
this function call is now deprecated and can be solved with v1 ffmpeg instead, possible examples here -- https://www.postman.com/sgp-labs/workspace/nca-toolkit-api/request/30756040-a7261c64-e7a1-4ca4-a224-24daa020c01a?action=share&creator=30756040&ctx=documentation
